packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -1,22 +1,30 @@
 // tslint:disable: no-string-literal
 import { readFileSync, writeFileSync } from 'fs';
+import { prompt as ask } from 'inquirer'
 
 const configFilesPath = `${__dirname}/resources/config`
 
+const dockerFilesPath = `${__dirname}/resources/docker`
+
 /**
  * Different database choices
  */
-export const defaultDB = {
-  name: 'postgres',
-  config: readFileSync(`${configFilesPath}/pgconfig.json`, 'utf8')
-}
-
-const dockerFilesPath = `${__dirname}/resources/docker`
 
-const pgDockerCompose = () => {
-  return readFileSync(`${dockerFilesPath}/pg-docker-compose.yml`, 'utf8')
+const getConfig = (database: string) => {
+  if(database === 'pg') {
```

Comment:
I would preffer postgress

Reply:
Kept it as `pg` since knex client going to use it, in display it's `PostgreSQL`